sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListenerSuite.scala

```diff
@@ -489,16 +501,17 @@ private case class MyPlan(sc: SparkContext, expectedValue: Long) extends LeafExe
 }
 
 
-class SQLListenerMemoryLeakSuite extends SparkFunSuite {
+class SQLAppStatusListenerMemoryLeakSuite extends SparkFunSuite {
 
-  // TODO: this feature is not yet available in SQLAppStatusStore.
-  ignore("no memory leak") {
+  test("no memory leak") {
     quietly {
       val conf = new SparkConf()
         .setMaster("local")
         .setAppName("test")
+        .set(LIVE_ENTITY_UPDATE_PERIOD, 0L) // Update the UI data immediately
         .set(config.MAX_TASK_FAILURES, 1) // Don't retry the tasks to run this test quickly
-        .set("spark.sql.ui.retainedExecutions", "50") // Set it to 50 to run this test quickly
+        // TODO: this feature is not yet available in SQLAppStatusStore.
+        // .set("spark.sql.ui.retainedExecutions", "50") // Set it to 50 to run this test quickly
```

Comment:
instead of totally disable this test because of an unimplemented feature, I'd like to still run it, but a little slower.

Reply:
This test is re-enabled in #19751.

Reply:
ok let me revert this part